lib/Mojolicious/Plugin/OpenAPI/Security.pm

```diff
@@ -1,19 +1,23 @@
 package Mojolicious::Plugin::OpenAPI::Security;
 use Mojo::Base -base;
 
+my %SECURITY_DEFINITION_PATH
+  = (2 => '/securityDefinitions', 3 => '/components/securitySchemes');
+
 sub register {
   my ($self, $app, $openapi, $config) = @_;
   my $handlers = $config->{security} or return;
 
-  return unless $openapi->validator->get('/securityDefinitions');
+  return unless $openapi->validator->get($SECURITY_DEFINITION_PATH{ $openapi->validator->version });
```

Comment:
The code is not run through perltidy. Please do so.

Reply:
I prefer naming which can be redundant but understandable in a glance. Matter of taste ))
Fixed.